templates/webapi/admin/user/index.html.ep

```diff
@@ -36,34 +89,42 @@
                                     %   $select = 2;
                                 % } elsif ($user->is_operator) {
                                 %   $select = 1;
-                % }
-                % $selects[$select] = Mojo::ByteStream->new("checked='checked' class='default'");
-                <input type="radio" name="role" value="user" id="<%= $user->id %>_user" <%= $selects[0] %>/>
-                <label for="<%= $user->id %>_user">
-                    <span class="fa-stack"> <!-- sexy checkboxes! -->
-                        <i class="far fa-circle fa-stack-1x"></i>
-                        <i class="fa fa-circle fa-stack-1x"></i>
-                    </span>
-                    User
-                </label>
-                <input type="radio" name="role" value="operator" id="<%= $user->id %>_operator" <%= $selects[1] %>/>
-                <label for="<%= $user->id %>_operator">
-                    <span class="fa-stack">
-                        <i class="far fa-circle fa-stack-1x"></i>
-                        <i class="fa fa-circle fa-stack-1x"></i>
-                    </span>
-                    Operator
-                </label>
-                <input type="radio" name="role" value="admin" id="<%= $user->id %>_admin" <%= $selects[2] %>/>
-                <label for="<%= $user->id %>_admin">
-                    <span class="fa-stack">
-                        <i class="far fa-circle fa-stack-1x"></i>
-                        <i class="fa fa-circle fa-stack-1x"></i>
-                    </span>
-                    Administrator
-                </label>
+                                % }
+                                % $selects[$select] = Mojo::ByteStream->new("checked='checked' class='default'");
+                                <input type="radio" name="role" value="user" id="<%= $user->id %>_user" <%= $selects[0] %>/>
+                                <label for="<%= $user->id %>_user">
+                                    <span class="fa-stack"> <!-- sexy checkboxes! -->
+                                        <i class="far fa-circle fa-stack-1x"></i>
+                                        <i class="fa fa-circle fa-stack-1x"></i>
+                                    </span>
+                                    User
+                                </label>
+                                <input type="radio" name="role" value="operator" id="<%= $user->id %>_operator" <%= $selects[1] %>/>
+                                <label for="<%= $user->id %>_operator">
+                                    <span class="fa-stack">
+                                        <i class="far fa-circle fa-stack-1x"></i>
+                                        <i class="fa fa-circle fa-stack-1x"></i>
```

Comment:
please work on finding a better approach for the indention to not have this many spaces in front.